.babelrc

```diff
@@ -2,7 +2,8 @@
   "presets": [
     ["@babel/env", {
       "targets": {
-        "browsers": ["last 2 versions"]
+        "browsers": ["last 2 versions"],
+        "esmodules": true
```

Comment:
Why it's added this?

Reply:
json-editor uses native js classes, and we extends these classes to make our customization.
Without this line, babel convert our classes into function that use `.call(this)` instead of class contructor.
This conversion causes a "Class constructor Client cannot be invoked without 'new'"
https://stackoverflow.com/questions/51860043/javascript-es6-typeerror-class-constructor-client-cannot-be-invoked-without-ne